docs/content/install.md

```diff
@@ -26,11 +26,11 @@ using the [rclone selfupdate](/commands/rclone_selfupdate/) command.
 
 To install rclone on Linux/macOS/BSD systems, run:
 
-    curl https://rclone.org/install.sh | sudo bash
+    sudo -v ; curl https://rclone.org/install.sh | sudo bash
```

Comment:
Maybe this should be

    sudo -v && curl ...

So we don't run the curl etc if the sudo fails?

Reply:
I'm happy to defer to your experience here - I'm not a shell scripting expert.

So you recommend we merge as-is with the `;` and experiment after?

Reply:
Yes, exactly. I also suspect that the behavior of `sudo &&` across platforms may not be consistent enough to rely on.

Random thought if there is a "Would be nice" task list:
"Create a test script that looks for some of the corner cases of sudo in this shell 'phrase'"

Reply:
OK I'll merge what we;ve got so far - thank you.

Doing cross platform experiments with shell scripts is tricky indeed since I don't have access to all the platforms we support!